### PR TITLE
gui-wm/hyprland: add 0.29.1

### PR DESCRIPTION
**Pull request by MrRoy**

### CODE REVIEW

**Comment by hotburger** [comment]
This version will have unexpected behavior without #32180, since it ships a config file to that requires xdg-desktop-portal >=1.17.0 and fixes a major usability issue with multiple incompatible portals installed.

File: gui-wm/hyprland/hyprland-0.29.0.ebuild [hunk 1]
```diff
@@ -75,10 +95,16 @@ pkg_setup() {
 src_prepare() {
 	if use video_cards_nvidia; then
 		cd "${S}/subprojects/wlroots" || die
-		eapply "${S}/nix/wlroots-nvidia.patch"
+		eapply "${S}/nix/patches/wlroots-nvidia.patch"
+		# https://bugs.gentoo.org/911597
+		# https://github.com/hyprwm/Hyprland/pull/2874
```
Comment (parona-source) [inline gui-wm/hyprland/hyprland-0.29.0.ebuild:100]: The pull request was merged so the sed line isnt needed anymore.

Comment (MrRoy) [reply gui-wm/hyprland/hyprland-0.29.0.ebuild]: Thanks, removed it

**Comment by MrRoy** [comment]
Updated the PR to 0.29.1

**Pull request closed without merging**